src/inbox/mod_inbox_rdbms_mssql.erl

```diff
@@ -40,8 +40,11 @@ set_inbox(Username, Server, ToBareJid, Content, Count, MsgId, Timestamp) ->
                             Timestamp :: non_neg_integer()) -> query_result().
 set_inbox_incr_unread(Username, Server, ToBareJid, Content, MsgId, Timestamp) ->
     Query = build_query(Username, Server, ToBareJid, Content, increment, MsgId, Timestamp),
-    mongoose_rdbms:sql_query(Server, Query).
-
+    mongoose_rdbms:sql_query(Server, Query),
+    mongoose_rdbms:sql_query(Server,
```

Comment:
What is the reason for returning the unread cont every time we increment the value? Is really needed for every increment?

Reply:
Yes, in order to update the value stored in accumulator.